<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AffairEditor</class>
 <widget class="QWidget" name="AffairEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>220</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="confirmButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>180</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华光淡古印_CNKI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>确  认</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LineEdit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>135</y>
     <width>300</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="affairLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>135</y>
     <width>115</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华光淡古印_CNKI</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>  事  件  ：</string>
   </property>
  </widget>
  <widget class="QTimeEdit" name="startTime">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>100</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="startLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>115</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华光淡古印_CNKI</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>开始时间：</string>
   </property>
  </widget>
  <widget class="QTimeEdit" name="endTime">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>78</y>
     <width>100</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="endLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>78</y>
     <width>115</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华光淡古印_CNKI</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>结束时间：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>180</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华光淡古印_CNKI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>删  除</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
